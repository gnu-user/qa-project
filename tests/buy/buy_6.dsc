TITLE
    Buy Event Title Exists (Invalid Title, contains invalid string)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the buy 
    transaction. When prompted, enter a title containing “END “. Verify that 
    the title is not accepted and an error occurs.
DEPENDENCIES
    buy_4
