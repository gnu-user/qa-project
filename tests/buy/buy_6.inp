login
administrator
buy
END OF THE WORLD!