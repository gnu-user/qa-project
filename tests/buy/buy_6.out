Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title:
Invalid event title, event title cannot contain the word "END".